components/schemas/environments/services/SchedulerEnvironmentService.yml

```diff
@@ -0,0 +1,28 @@
+---
+title: SchedulerEnvironmentService
+type: object
+nullable: true
```

Comment:
do we use nullable in the other service schema roots? i try to avoid this, because if i target this type specifically, it'll be nullable as part of that.